lib/lotus/generators/database_config.rb

```diff
@@ -0,0 +1,88 @@
+module Lotus
+  module Generators
+    class DatabaseConfig
+      SUPPORTED_ENGINES = {
+        'mysql'      => { type: :sql,         mri: 'mysql',   jruby: 'jdbc-mysql'    },
```

Comment:
no, on second look, it seems to me we do support that, please ignore my comment above